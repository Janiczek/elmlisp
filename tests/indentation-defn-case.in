(defn update : (-> Msg Model Model)
 (msg model)
 (case msg
   (Inc (+ 1 model))
   (Dec (- 1 model))))
