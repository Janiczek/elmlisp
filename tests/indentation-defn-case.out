update : Msg -> Model -> Model
update msg model =
    case msg of
        Inc ->
            1 + model

        Dec ->
            1 - model
